src/App.tsx

```diff
@@ -1,16 +1,27 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState(moviesFromServer);
+
+  const onAdd = (movie: Movie) => {
+    setMovies(current => [
```

Comment:
Better use `prevState` variable in this case
```suggestion
    setMovies(prevState => [
```